package-res/reportviewer/reportviewer.js

```diff
@@ -1516,6 +1544,7 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
         var me = this;
         //Update current values
         me._currentReportStatus = mainJobStatus.status;
+        _currentReportStatus = mainJobStatus.status;
```

Comment:
If this line is needed, then you are accessing "_currentReportStatus" somewhere without qualifying it with the "this" or "me" reference. Fix your new "renderBackgroundBtn" function and qualify access with "this", and therefore access the correct member variable of your current object..